docs/src/modules/java/pages/views.adoc

```diff
@@ -452,15 +456,116 @@ Use the following filter predicates to further refine results:
 * `<` less than
 * `\<=` less than or equals
 
-Combine filter conditions with the `AND` and `OR` operators.
+==== Logical operators
+
+Combine filter conditions with the `AND` and `OR` operators, and negate using the `NOT` operator. Group conditions using parentheses. Note that `AND` has precedence over `OR`.
 
 [source,proto,indent=0]
 ----
 SELECT * FROM customers WHERE
   name = :customer_name AND address.city = 'New York' OR
-  name = :customer_name AND address.city = 'San Francisco'
+  NOT (name = :customer_name AND address.city = 'San Francisco')
+----
+
+NOTE: `OR` support is currently disabled, until it can be more efficiently indexed.
```

Comment:
maybe move this to the top of the section?

Reply:
Yeah, this one is awkward at the moment. Might be good to comment out the full text, and have something that doesn't demonstrate OR as well. I'll update.

Reply:
Removed OR. Commented out the original.